tests/cli/test_cli_main.py

```diff
@@ -5,81 +5,113 @@
 import pytest
 
 from brownie._cli.__main__ import main as cli_main
+from brownie._cli.console import Console as cli_console
 
 
 @pytest.fixture
-def cli_tester(monkeypatch):
-    c = CliTester(monkeypatch)
-    yield c
-    c.close()
+def cli_tester(monkeypatch, mocker):
+    tester = CliTester(monkeypatch, mocker)
+
+    yield tester
+    tester.close()
 
 
 class CliTester:
-    def __init__(self, monkeypatch):
+    def __init__(self, monkeypatch, mocker):
         self.argv = sys.argv.copy()
         self.monkeypatch = monkeypatch
-        self.called = False
-        self.total = 0
-        self.count = 0
-
-    def set_target(self, target):
-        self.monkeypatch.setattr(target, self.catch)
-
-    def set_subtargets(self, *targets):
-        for item in targets:
-            self.monkeypatch.setattr(item, self.incremental_catch)
-            self.total += 1
+        self.mocker = mocker
 
-    def run(self, argv, args=(), kwargs={}):
+    def run_and_test_parameters(self, argv, parameters={}):
         sys.argv = ["brownie"] + argv.split(" ")
-        self.args = args
-        self.kwargs = kwargs
         cli_main()
-        assert self.called is True
-        assert self.count == self.total
-        self.called = False
-        self.count = 0
+        assert self.mock_subroutines.call_args == parameters
 
-    def catch(self, *args, **kwargs):
-        assert self.args == args
-        assert self.kwargs == kwargs
-        self.called = True
-
-    def incremental_catch(self, *args, **kwargs):
-        self.count += 1
+    def mock_subroutines(self, *args, **kwargs):
+        print("Monkey Patched Function")
 
     def close(self):
         sys.argv = self.argv
 
 
 def test_cli_init(cli_tester):
-    cli_tester.set_target("brownie.project.new")
-    cli_tester.run("init", args=(".", False))
-    cli_tester.run("init test/path --force", args=("test/path", True))
+    cli_tester.mocker.spy(cli_tester, "mock_subroutines")
```

Comment:
this call is repeated in every test - you could incude it at the start of `cli_tester.run_and_test_parameters`

Reply:
Added the statement as a part of the fixture.